bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-resources",
-  "version": "2.0.5",
+  "version": "2.1.0",
```

Comment:
2.0.6 would be fine here, we normally only change the minor version number when we do a pretty big re-write such as we did with Slider or Matching recently